jupyter/README.md

```diff
@@ -0,0 +1,417 @@
+
+Now to see what the circuit does when under qsim. Again, copy the following into your VM window:
+
+
+```
+    simulator = qsimcirq.QSimSimulator()
+    result = simulator.simulate(circuit)
+    print("Result:")
+    print(result)
+```
+
+
+The output will be:
+
+
+```
+    measurements: (no measurements)
+    output vector: (0.5+0.5j)|0⟩ + (0.5-0.5j)|1⟩
+```
+
+
+You have successfully simulated a quantum circuit on Google Cloud Platform using a Singularity container.
+
+
+### Running your own script {#running-your-own-script}
+
+If you want to run a Python script, you can locate a file in the home directory on your VM, then run something like in the container shell
+
+	`$ python3 /homedir/myscript.py`
+
+
+### Exit the container {#exit-the-container}
+
+Exit the container by typing cntl-d twice. You will see the output like:
+
+
+```
+    [root@79804d33f250 /]# exit
+```
+
+
+-----
+
+
+## **Cleaning up** {#cleaning-up}
+
+_[The cleaning up section is required. Use it to ensure they don’t get billed for any extra time.]_
+
+To avoid incurring charges to your Google Cloud Platform account for the resources used in this tutorial:
+
+
+### Delete the VM {#delete-the-vm}
+
+It is usually recommended to delete the entire project you created for this tutorial, but if you want to continue using the project, you can easily either STOP the VM or Delete it. Select the VM, the either select the STOP square, or the DELETE trash can.
+
+
+
+
+![alt_text](images/image9.png "image_tooltip")
+
+
+
+### Delete the project {#delete-the-project}
+
+The easiest way to eliminate billing is to delete the project you created for the tutorial.
+
+**Caution**: Deleting a project has the following effects:
+
+
+
+    *   **Everything in the project is deleted.** If you used an existing project for this tutorial, when you delete it, you also delete any other work you've done in the project.
+    *   **Custom project IDs are lost.** When you created this project, you might have created a custom project ID that you want to use in the future. To preserve the URLs that use the project ID, such as an **<code>appspot.com</code></strong> URL, delete selected resources inside the project instead of deleting the whole project.
+
+    If you plan to explore multiple tutorials and quickstarts, reusing projects can help you avoid exceeding project quota limits.
+
+1. In the Cloud Console, go to the **Manage resources** page. \
+[Go to the Manage resources page](https://console.cloud.google.com/iam-admin/projects)
+2. In the project list, select the project that you want to delete and then click **Delete **
+
+
+![alt_text](images/image3.png "image_tooltip")
+.
+3. In the dialog, type the project ID and then click **Shut down** to delete the project.
+
+----
+
+
+## **What's next** {#what's-next}
+
+Additional tutorial examples are available here:
+
+
+
+*   [https://cirq.readthedocs.io/en/stable/examples.html](https://cirq.readthedocs.io/en/stable/examples.html)
```

Comment:
These links should have names, e.g. `Cirq Examples` and `Getting started with qsimcirq`.